cmd/earthly/flags.go

```diff
@@ -217,6 +217,14 @@ func (app *earthlyApp) rootFlags() []cli.Flag {
 			Destination: &app.buildID,
 			Hidden:      true, // Internal.
 		},
+		&cli.IntFlag{
```

Comment:
Can we use a duration instead?
```suggestion
		&cli.DurationFlag{
```